bedrock/configuration-files.md

```diff
@@ -11,13 +11,17 @@ published: true
 ---
 The root `web/wp-config.php` is required by WordPress and is only used to load the other main configs. Nothing else should be added to it.
 
-`config/application.php` is the main config file that contains what `wp-config.php` usually would. Base options should be set in there.
+The file to modify for configuration options is `config/application.php`. This file that contains what `wp-config.php` usually would.
 
-For environment specific configuration, use the files under `config/environments`. By default there are `development`, `staging`, and `production` but these can be whatever you require.
+These base configuration options should be production-standard _SAFE SETTINGS_ and will be used in all environments except where specifically overridden.
 
-The environment configs are required **before** the main `application` config so anything in an environment config takes precedence over `application`.
+For environment-specific configuration option **overrides**:
 
-Note: You can't re-define constants in PHP. So if you have a base setting in `application.php` and want to override it in `production.php` for example, you have a few options:
+  * Use or create files under `config/environments`. The `config/application.php` code will require any file in the `config/environments` directory with a filename matching the `WP_ENV` assignment. This will either be in a `.env` file at the top of the Bedrock installation or for a Trellis installation, defined in the `wordpress_sites` settings.
```

Comment:
Not sure about the wording here regarding `WP_ENV`. What about something like:

* Use an existing environment config in `config/environments` or create a new one. Bedrock will require any file in the `config/environments` directory with a filename matching the `WP_ENV` environment variable. This environment variable can be set in a few ways:
  * in the `.env` file as described in our [installation docs](https://roots.io/bedrock/docs/installing-bedrock/#installation)
  * via [Trellis config](https://roots.io/trellis/docs/wordpress-sites/) if you're using Trellis
  * or as a last resort, hardcoding it in `config/application.php`

Reply:
Love it. What about "Bedrock will `require` any"?

Reply:
Ha, I meant the term generically but that's a good idea. Works in both cases 👍